tools/c7n_org/tests/test_org.py

```diff
@@ -60,6 +75,42 @@ def setup_run_dir(self, accounts=None, policies=None):
         os.makedirs(cache_path)
         return root
 
+    def test_validate_azure_provider(self):
+        run_dir = self.setup_run_dir(
+            accounts=ACCOUNTS_AZURE,
+            policies={'policies': [{
+                'name': 'vms',
+                'resource': 'azure.vm'}]
+            })
+        logger = mock.MagicMock()
+        run_account = mock.MagicMock()
+        run_account.return_value = ({}, True)
+        self.patch(org, 'logging', logger)
+        self.patch(org, 'run_account', run_account)
+        self.change_cwd(run_dir)
+        runner = CliRunner()
+        result = runner.invoke(
+            org.cli,
+            ['run', '-c', 'accounts.yml', '-u', 'policies.yml',
+             '--debug', '-s', 'output', '--cache-path', 'cache'],
+            catch_exceptions=False)
+        self.assertEqual(result.exit_code, 0)
+
+    def test_validate_gcp_provider(self):
```

Comment:
this should really be parameterized with pytest

Reply:
switching to pytest parameterize actually results in a significant diff increase as we need to switch out from our existing usage of unittest classes to get pytests parameterize support.